modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/test/js/components/card-shortcut/CardShortcut.es.js

```diff
@@ -0,0 +1,138 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {act, cleanup, fireEvent, render} from '@testing-library/react';
+import React from 'react';
+
+import CardShortcut from '../../../../src/main/resources/META-INF/resources/js/components/card-shortcut/CardShortcut.es';
+import SidebarContextProviderWrapper from '../../SidebarContextProviderWrapper.es';
+
+const fields = [
+	{
+		label: 'Field 1',
+		name: 'field1',
+		options: {Option: 'Option'},
```

Comment:
I put it because it belongs to the structure of every field, and I've copied from another test file. But there's no need in this test case. I can remove it.
FYI, the current structure of every field is like:
```
columns: {}
label: "Text"
name: "Text"
options: {Option: "Option"}
rows: {}
type: "text"

```